--- a/SPOUTSDK/SpoutGL/SpoutGL.cpp	2025-10-22 03:05:05.000000000 +0200
+++ b/SPOUTSDK/SpoutGL/SpoutGL.cpp	2026-08-11 10:31:09.777943200 +0200
@@ -1385,7 +1385,7 @@
 {
 	HANDLE hInteropObject = nullptr;
 	DWORD dwError = 0;
-	char tmp[128]={};
+	char tmp[256]={};
 
 	SpoutLogNotice("spoutGL::LinkGLDXtextures - device 0x%X, texture 0x%X, GL texture %d",
 		PtrToUint(pDXdevice), PtrToUint(pSharedTexture), glTexture);
@@ -1415,25 +1415,25 @@
 	// Report the error if wglDXOpenDeviceNV failed
 	if (!m_hInteropDevice) {
 		dwError = GetLastError();
-		sprintf_s(tmp, 128, "spoutGL::LinkGLDXtextures : wglDXOpenDeviceNV(0x%.7X) no Interop device - error %lu (0x%.X)\n",
+		sprintf_s(tmp, 256, "spoutGL::LinkGLDXtextures : wglDXOpenDeviceNV(0x%.7X) no Interop device - error %lu (0x%.X)\n",
 			LOWORD(pDXdevice), dwError, LOWORD(dwError));
 		// Other errors reported
 		// 1008, 0x3F0 - ERROR_NO_TOKEN
 		switch (LOWORD(dwError)) {
 		case 0:
-			strcat_s(tmp, 128, "    No error");
+			strcat_s(tmp, 256, "    No error");
 			break;
 		case ERROR_OPEN_FAILED:
-			strcat_s(tmp, 128, "    Could not open the Direct3D device.");
+			strcat_s(tmp, 256, "    Could not open the Direct3D device.");
 			break;
 		case ERROR_NOT_SUPPORTED:
 			// This can be caused either by passing in a device from an unsupported DirectX
 			// version, or by passing in a device referencing a display adapter that is
 			// not accessible to the GL.
-			strcat_s(tmp, 128, "    The dxDevice is not supported.");
+			strcat_s(tmp, 256, "    The dxDevice is not supported.");
 			break;
 		default:
-			strcat_s(tmp, 128, "    Unknown error.");
+			strcat_s(tmp, 256, "    Unknown error.");
 			break;
 		}
 		SpoutLogError("%s", tmp);
@@ -1466,23 +1466,23 @@
 		// Error codes are 32-bit values, but expected results are in the low word.
 		// 006E is ERROR_OPEN_FAILED (110L)
 		dwError = GetLastError();
-		sprintf_s(tmp, 128, "spoutGL::LinkGLDXtextures - wglDXRegisterObjectNV :error %u, (0x%.X)\n",
+		sprintf_s(tmp, 256, "spoutGL::LinkGLDXtextures - wglDXRegisterObjectNV :error %u, (0x%.X)\n",
 			LOWORD(dwError), LOWORD(dwError));
 		switch (LOWORD(dwError)) {
 		case 0:
-			strcat_s(tmp, 128, "    No error");
+			strcat_s(tmp, 256, "    No error");
 		break;
 		case ERROR_INVALID_HANDLE:
-			strcat_s(tmp, 128, "    No GL context is current.");
+			strcat_s(tmp, 256, "    No GL context is current.");
 			break;
 		case ERROR_INVALID_DATA:
-			strcat_s(tmp, 128, "    Incorrect GL name, type or access parameters.");
+			strcat_s(tmp, 256, "    Incorrect GL name, type or access parameters.");
 			break;
 		case ERROR_OPEN_FAILED:
-			strcat_s(tmp, 128, "    Failed to open the Direct3D resource.");
+			strcat_s(tmp, 256, "    Failed to open the Direct3D resource.");
 			break;
 		default:
-			strcat_s(tmp, 128, "    Unknown error.");
+			strcat_s(tmp, 256, "    Unknown error.");
 			break;
 		}
 		SpoutLogError("%s", tmp);
